File: TuristickaAgencija.Model/ListaZelja.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TuristickaAgencija.Model
{
    public class ListaZelja
    {
        public int Id { get; set; }
        public int PutovanjeId { get; set; }
        public int KorisnikId { get; set; }
        public string Opis { get; set; }
    }
}

File: UI/turisticka_agencija_admin/lib/models/ListaZelja.dart
import 'package:json_annotation/json_annotation.dart';

part 'ListaZelja.g.dart';

@JsonSerializable()
class ListaZelja {
    int id;
    int korisnikId;
    String opis;
    int putovanjeId;

    ListaZelja({required this.id,required this.korisnikId,required this.opis,required this.putovanjeId});

    factory ListaZelja.fromJson(Map<String, dynamic> json) => _$ListaZeljaFromJson(json);

    Map<String, dynamic> toJson() => _$ListaZeljaToJson(this);
}